vendor/composer/autoload_commands_classmap.php

```diff
@@ -6,4 +6,100 @@
 $baseDir = dirname($vendorDir);
 
 return array(
+    'CLI_Command' => $vendorDir . '/wp-cli/wp-cli/php/commands/src/CLI_Command.php', 
```

Comment:
Should we be pushing this into customers?

Reply:
No, that's been added due to a dev-including autoload dump generation, fixed.